src/bun.js/test/Order.zig

```diff
@@ -139,15 +142,20 @@ pub fn generateOrderTest(this: *Order, current: *ExecutionEntry) bun.JSError!voi
 
     // set skip_to values
     var index = list.first;
-    var skip_to = current.next;
+    var skip_to_next: ?*ExecutionEntry = current;
```

Comment:
`skip_to_next` is a bit of a confusing name. If `skip_to` was `current.next`, I would expect something called `skip_to_next` to be `skip_to.next`, or `current.next.next`. But it's actually the opposite.

Reply:
renamed to 'failure_skip_past' although that's still somewhat confusing